### PR TITLE
Building highlight

### LINKED ISSUE #47: [Task 1.4.4] Highlight User’s Current Building on Map

### PR DESCRIPTION
**Pull request by simonbern**
## What changed
-The building a user is located in will be highlighted in blue

## Why
-So a user can easily know in which building he or she is located in

## How to test
- [ ] mobile: npm test BuildingPolygon.test.tsx
- [ ] server: npm test (or npm run test:coverage)

## Linked issue
Closes #47 

## Review checklist
- [ ] code is readable
- [ ] tests updated/added
- [ ] edge cases handled

### CODE REVIEW

File: mobile/__tests__/Buildingpolygon.test.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (grylajoie) [reply mobile/__tests__/Buildingpolygon.test.tsx]: File name not consistent, rework

[the remark this answers is not in the document]
Comment (grylajoie) [reply mobile/__tests__/Buildingpolygon.test.tsx]: Buildingpolygon.test.tsx is TypeScript file but not the component.
